multinet/api.py

```diff
@@ -100,6 +102,23 @@ def get_table_rows(workspace: str, table: str, offset: int = 0, limit: int = 30)
     return Workspace(workspace).table(table).rows(offset, limit)
 
 
+@bp.route("/workspaces/<workspace>/tables/<table>/metadata", methods=["GET"])
+@require_reader
+@swag_from("swagger/get_metadata.yaml")
+def get_table_metadata(workspace: str, table: str) -> Any:
+    """Retrieve the metadata of a table, if it exists."""
+    metadata = Workspace(workspace).table(table).get_metadata()
+    return "" if metadata is None else metadata.dict()
```

Comment:
Since this returns a `dict` when there is metadata, shouldn't it return `{}` instead of `""` when there isn't any?

Reply:
I'd say you're right, but this made me think of something; How should we handle existing tables? In the current implementation, any new table will at least have empty metadata, but this does nothing for existing tables. I think it would make sense to ensure that this metadata exists, by handling the case where it doesn't right in `get_metadata`. So the current case where it returns `None`, would instead initialize the metadata as empty, and return that.

Reply:
This issue would cover that right? https://github.com/multinet-app/multinet-server/issues/405

Reply:
Yes but it would be good to add this particular migration to that issue.